src/lib.rs

```diff
@@ -174,9 +174,11 @@
 )]
 #![allow(clippy::type_complexity)]
 
-use std::sync::{atomic, Arc, Mutex};
+mod sync;
 
-type Epochs = Arc<Mutex<slab::Slab<Arc<atomic::AtomicUsize>>>>;
```

Comment:
Question - how do we feel about changing `atomic::X` imports to just be `X`? Do we want to maintain the module for readability/discoverability?

Reply:
That seems fine to me!